### ISSUE TITLE
Training details for v1.8.0 or model version 10

### ISSUE THREAD

**Issue by isseebx123** [lightvector/KataGo#415]
Thank you for the great experiments. I have two questions.

Did the version 10 model improve performance by training the self-play data created by the version 8 model? I am wondering what data was used to train the version 10 model.

I'm wondering if you used a high learning rate to train version 10 model, and if the current learning rate is using a low value.

Sorry if you have already explained it elsewhere, or if you didn't understand my question because I'm not good at English.

**Comment by lightvector** [comment]
Exactly the same data as the version 8 model. In fact, the version 10 models that kata1 started from were the exact same as the version 8 models, just updated to the new model format with a small adjustment to the structure and targets for one of the auxiliary losses.

In general, KataGo's model version has nothing to do with learning rate, data, or the overall training method. The point of "version" is to distinguish when models are incompatible with one another when loading the model from disk. So in this case, what changed was a detail of the training that should make no difference for users (yet!), but made the model format incompatible.

**Comment by isseebx123** [comment]
Sorry for the late response. Thank you for the reply.

> In general, KataGo's model version has nothing to do with learning rate, data, or the overall training method. The point of "version" is to distinguish when models are incompatible with one another when loading the model from disk. So in this case, what changed was a detail of the training that should make no difference for users (yet!), but made the model format incompatible.

Yes. I already understand what you said. What I wanted to say is that the training of the previous version model dropped the learning rate, but I was curious about the learning rate of the current training. It can be seen from the table below that the learning rate has been dropped.

![image](https://user-images.githubusercontent.com/26362491/107896325-4c7e3e80-6f79-11eb-96f9-7b0c287e9f5e.png)

In other words, I wonder if KataGo is still using the low learning rate you used last time.

Thank you.

**Comment by lightvector** [comment]
Ah, I sorry misunderstood you. Yes, the new "kata1" run is using the exact same learning rate that was used at the end of the g170 run. I don't plan to change it any time soon, since it seems the run is progressing just fine.

Terminology-wise, the word you're probably looking for is "run". The "g104" run is the first clearly superhuman-level run, which was featured in KataGo's published paper. The "g170" run was the one that ended in June 2020, and the "kata1" run is the public run ongoing now. The word "version" is only used to refer to KataGo's software releases (e.g. "version 1.8.0"). Or to the model format on disk if for some reason you care about the disk format and internal architecture details but which has not much to do with other things.

I hope that makes sense! :)

**Comment by lightvector** [comment]
And to clarify, the reason why there's a distinction, is because the "kata1" run is obviously not going to suddenly end once we have version 1.8.1, or if we upgrade the model format again - we're going to keep training of course - so it makes sense to think of the run itself existing separately from the versions of anything else.

**Comment by isseebx123** [comment]
Thanks for the kind and quick response.
